### ISSUE TITLE
https://ticker.openbazaar.org/api behind cloudflare: tor

### ISSUE THREAD

**Issue by mocacinno** [OpenBazaar/openbazaar-go#1446]
I tried to setup the latest release of openbazaar over tor (tor only, NOT dual stack). Everything works fine, except the error 
invalid character '<' looking for beginning of value

In the console output of openbazaar-go.

In the ob.log, i found following line:
09:20:42.861 [ERROR] [exchangeRates/fetch] Failed to decode JSON from https://ticker.openbazaar.org/api invalid character '<' looking for beginning of value

After manually calling the api with curl:
curl -x socks5h://127.0.0.1:9050/ https://ticker.openbazaar.org/api

I realised  noticed following output:
~ snip
<title>Attention Required! | Cloudflare</title>  
~ snip

In other words, the ticker api is behind cloudflare, and tor-only openbazaar nodes cannot reach it, thus are unable to parse the output, thus show errors in their log and console output

**Comment by hoffmabc** [comment]
Disabled tor proxy please try again @mocacinno

**Comment by mocacinno** [comment]
Sorry for the slow response, was AFK during the weekend.
I did test without proxy and it worked fine... At the moment it's also working with proxy after i changed my circuit and got a new exit node. I have no idear if the cloudflare settings for the api were changed or if i just got lucky and have a new exit node cloudflare hasn't blocked yet

**Comment by placer14** [comment]
Yes, we disabled the tor proxy. If the problem continues, please reopen this issue.
